pyproject.toml

```
@@ -0,0 +1,141 @@
+# For more information about configuring project metadata for the
+# setuptools build backend, please see
+# https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools"]
+
+[project]
+authors = [
+    { name = "Cybersecurity and Infrastructure Security Agency", email = "github@cisa.dhs.gov" }
+]
+# See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+classifiers = [
+    # How mature is this project? Common values are
+    #   3 - Alpha
+    #   4 - Beta
+    #   5 - Production/Stable
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    # Indicate who your project is intended for
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    # Specify the Python versions you support here. In particular, ensure
+    # that you indicate whether you support Python 2, Python 3 or both.
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    # "Programming Language :: Python :: 3.11",
+    # "Programming Language :: Python :: 3.12",
+    # "Programming Language :: Python :: 3.13",
+    # "Programming Language :: Python :: 3.14",
+    "Programming Language :: Python :: Implementation :: CPython",
+]
+dependencies = [
+    "docopt>=0.6.2",
+    "publicsuffixlist[update]>=0.9.2 ",
+    "pyopenssl>=17.5.0",
+    "pytablereader>=0.15.0",
+    "pytablewriter>=0.27.2",
+    "python-dateutil>=2.7.3",
+    "pytz>=2018.5",
+    "requests>=2.18.4",
+    "sslyze>=3.0.0,<5.0.0",
+    "wget>=3.2",
+]
+description = "Scan websites for HTTPS deployment best practices"
+dynamic = ["readme", "version"]
+keywords = ["https best practices"]
+# Although PEP 639 allows us to specify the license via a simple SPDX
+# string, the versions of setuptools we have to use for Python 3.7 and
+# 3.8 do not allow this; hence, this antiquated format from the
+# deprecated PEP 621.
+license = { text = "CC0-1.0" }
+name = "pshtt"
+requires-python = ">=3.7"
+
+[project.optional-dependencies]
+# IMPORTANT: Keep type hinting-related dependencies of the dev section
+# in sync with the mypy pre-commit hook configuration (see
+# .pre-commit-config.yaml). Any changes to type hinting-related
+# dependencies here should be reflected in the additional_dependencies
+# field of the mypy pre-commit hook to avoid discrepancies in type
+# checking between environments.
+dev = [
+    "types-docopt",
+    "types-pyOpenSSL",
+    "types-requests",
+    "types-urllib3",
+]
+test = [
+    "coverage",
+    "coveralls",
+    "pre-commit",
+    "pytest-cov",
+    "pytest",
+]
+
+[project.scripts]
+example = "pshtt.cli:main"
```

Use correct script name:
```suggestion
pshtt = "pshtt.cli:main"
```